PROTOCOL.md

```diff
@@ -304,6 +304,9 @@ its own.
 
 The `MAIR_EL1` register contents are described above, in the caching section.
 
+For base revisions &gt; 0, `TCR_EL1.tg0` is set to a 4KiB granule size,
+`TCR_EL1.t0sz` is set to 16, and `TTBR0_EL1` is left undefined.
```

Comment:
A little lower the spec says:
> The used translation granule size is undefined.

IMO it'd be better to place this next to that. Also IMO tg0/t0sz should be uppercase (as it is in the ARMARM) but that's not a big deal.

Reply:
Is that better?